.github/ISSUE_TEMPLATE/tracking_issue.md

```diff
@@ -0,0 +1,17 @@
+---
+name: Tracking Issue
+about: Track the progress of a specific task or feature
+title: 'Short summary of the task or feature'
+labels: 'C-tracking-issue'
```

Comment:
I am wondering why the text needs to be wrapped between ' and ' just for the labels field, but not for name and about.